TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Positioning/PositionSystem.java

```diff
@@ -0,0 +1,79 @@
+package org.firstinspires.ftc.teamcode.Positioning;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+import org.firstinspires.ftc.teamcode.Components.DriveTrain;
+import org.firstinspires.ftc.teamcode.Components.Gyro;
+import org.firstinspires.ftc.teamcode.Positioning.Utils.Vector2d;
+
+@Deprecated //INCOMPLETE AND UNSAFE!!!
+public class PositionSystem {
+
+    public enum TgtPositions{
+        DEPOT(new Vector2d(0,0)), CRATER(new Vector2d(0,0));
+
+        private final Vector2d tgtPosition;
+        TgtPositions(Vector2d id) { this.tgtPosition = id; }
+        public Vector2d getValue() { return tgtPosition; }
+    }
+
+    OpMode opMode;
+    DriveTrain driveTrain;
+    Gyro gyro;
+
+    private final double inchPerTick = 0.0D; //TODO: GET RATIO
+    private double oldSec;
+    private double oldDistance;
+    private double oldHeading;
+    private Vector2d position;
+
+    public PositionSystem(OpMode opMode, Gyro gyro, DriveTrain driveTrain){
+        this.opMode = opMode;
+        this.gyro = gyro;
+        this.driveTrain = driveTrain;
+    }
+
+    public void init(boolean debug){
+        if(debug){
+            opMode.telemetry.addData("INFO", "Pos Sys INIT");
+        }
+    }
+
+    public void updatePos(){
+        double pastSec = System.currentTimeMillis()/1000.0-oldSec;
+        if(pastSec > 0.25) {
```

Comment:
why are you waiting 0.25 seconds, seems like there could be fairly large inaccuracies behind this

Reply:
@GarrettBurroughs It was to from the frc version that helped prevent the issues with the tcpip log, and the overall speed. I can have made it 1/16 of a sec but for now im gonna keep it in.